vendors/cypress/bluetooth/include/wiced_bt_nvram_access.h

```diff
@@ -170,4 +170,4 @@ wiced_result_t wiced_bt_nvram_access_register(wiced_bt_nvram_access_t *p_nvram_a
 
 #ifdef __cplusplus
 }
-#endif
+#endif
```

Comment:
It looks like there are extra characters at the end of this line, can you remove them?

Reply:
Thanks. Will fix.